easybuild/easyconfigs/s/spectral.methods/spectral.methods-0.7.2.133-intel-2018a-R-3.4.4.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'Bundle'
+
+name = 'spectral.methods'
+version = '0.7.2.133'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://cran.r-project.org/web/packages/spectral.methods'
+description = """Contains some implementations of Singular Spectrum Analysis (SSA) for the gapfilling and
+ spectral decomposition of time series."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+dependencies = [
+    ('R', '3.4.4', '-X11-20180131'),
+    ('UDUNITS', '2.2.26'),
+]
+
+exts_defaultclass = 'RPackage'
+exts_default_options = {
+    'source_urls': [
+        'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'http://cran.r-project.org/src/contrib/',  # current version of packages
+        'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_list = [
+    ('RNetCDF', '1.9-1'),
+    ('ncdf.tools', '0.7.1.295'),
+    (name, version),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': [name],
+}
+
+modextrapaths = {'R_LIBS': ''}
```

Comment:
This is redundant, as it is updated in the easyblock. The same for a bunch of other `RPackage` easyconfigs. Tackle them in a different PR?